types/test.d.ts

```diff
@@ -1105,6 +1126,11 @@ export type PlaywrightWorkerArgs = {
    * Browser instance, shared between multiple tests.
    */
   browser: Browser;
+
+  /**
+   * URL of the web server. For example, http://localhost:1234
+   */
+  baseURL: String;
```

Comment:
- I do think this should go to the PlaywrightWorkerOptions.
- I would also expect my `page` to be pre-navigated to this url? Perhaps we should not rush with this one, and expose something like `webServer: string` fixture instead?